tokens/PLU.json

```diff
@@ -14,5 +15,62 @@
     "chainId": 98866,
     "decimals": 18,
     "logoURI": "https://assets.plume.org/images/logos/pETH/pETH-token.svg"
+  },
+  {
+    "name": "Nest WisdomTree Vault",
+    "address": "0x29bF22381A5811deC89dC7b46A5Ce57aD02c0240",
+    "symbol": "nWISDOM",
+    "chainId": 98866,
+    "decimals": 6,
+    "logoURI": "https://assets.plume.org/images/logos/nest/nWISDOM/nWISDOM-token.svg"
+  },
+  {
+    "name": "Nest Alpha Vault",
+    "address": "0x593cCcA4c4bf58b7526a4C164cEEf4003C6388db",
+    "symbol": "nALPHA",
+    "chainId": 98866,
+    "decimals": 6,
+    "logoURI": "https://assets.plume.org/images/logos/nest/nALPHA/nALPHA-token.svg"
+  },
+  {
+    "name": "Nest Basis Vault",
+    "address": "0x0000000000000000000000000000000000000000",
+    "symbol": "nBASIS",
+    "chainId": 98866,
+    "decimals": 6,
+    "logoURI": "https://assets.plume.org/images/logos/nest/nBASIS/nBASIS-token.svg"
+  },
+  {
+    "name": "Matrixdock Gold Vault",
+    "address": "0xA0C4F78A29ead4ABf6b7f5B3F0d05C0f3EAb8DDf",
+    "symbol": "XAUM",
+    "chainId": 98866,
+    "decimals": 18,
+    "logoURI": "https://assets.plume.org/images/logos/Matrixdock/XAUm-token.svg"
+  },
+  {
+    "name": "USDC Native",
```

Comment:
why not call it `USDC`?

Reply:
yeah just conferred w/ product on our end, plain USDC is preferred. changing!